bindings/c/test/mako/mako.c

```diff
@@ -940,6 +1093,17 @@ int worker_process_main(mako_args_t* args, int worker_id, mako_shmhdr_t* shm) {
 
 	for (i = 0; i < args->num_threads; i++) {
 		thread_args[i].thread_id = i;
+		for (int j = 0; j < MAX_OP; j++) {
+			thread_args[i].block[j] = (lat_block_t*)malloc(sizeof(lat_block_t));
```

Comment:
Why do we need to allocate the stats buffer here and pass around `is_memory_allocated` everywhere?

Reply:
This is the initial memory allocation to each operation for each thread per process. is_memory_allocated is just an indicator whether the heap memory was allocated or not at any point of time for that specific operation.